### PR TITLE
Fix false positive in test tflite2

### PR DESCRIPTION
**Pull request by DeepthiYV**
Test tflite2 was showing false positive even with the error "no module named numpy".  Now Test is modified to installs required modules and verifies the output after the test run. 
While fixing the test, I found a bug in tensorflow package installation and bug is reported. Now I added softfail  in the test.
https://bugzilla.opensuse.org/show_bug.cgi?id=1199330
https://bugzilla.opensuse.org/show_bug.cgi?id=1199429

- Related ticket: https://progress.opensuse.org/issues/110548
- Needles: No
- Verification run: 
   Tumbleweed: [x86_64](https://openqa.opensuse.org/tests/2353183#step/tflite2/46)  | [aarch64](https://openqa.opensuse.org/tests/2352999#step/tflite2/34)
    Leap:  [x86_64](https://openqa.opensuse.org/tests/2330104#step/tflite2/31) | [aarch64](https://openqa.opensuse.org/tests/2353188#step/tflite2/19)

### CODE REVIEW

File: tests/ai_ml/tflite2.pm [hunk 1]
```diff
@@ -28,8 +30,16 @@ sub run {
...
-    assert_script_run("python3 ~/data/ai_ml/label_image.py --image ~/data/ai_ml/images/White_shark.jpg --model_file mobilenet_v1_1.0_224_quant.tflite --label_file labels_mobilenet_quant_v1_224.txt | tee /dev/$serialdev | head -n1", sub { m/great white shark$/ });
+    my $result = script_output("python3 ~/data/ai_ml/label_image.py --image ~/data/ai_ml/images/White_shark.jpg --model_file mobilenet_v1_1.0_224_quant.tflite --label_file labels_mobilenet_quant_v1_224.txt | tee /dev/$serialdev | head -n1", proceed_on_failure => 1);
+    record_info("TEST LOG", "$result");
+
+    if ($result !~ /great white shark/) {
+        if ($ret == 4 && is_opensuse) {
+            record_soft_failure("boo#1199429 nothing provides 'libprotobuf.so.30()(64bit)' needed by the to be installed tensorflow2");
+        } elsif ($ret == 104 && is_leap) {
+            record_soft_failure("boo#1199330 package python3-tensorflow is not found");
+        }
```
Comment (mloviska) [inline tests/ai_ml/tflite2.pm:41]: Looks nice! Just a minor note that caught my attention, if the result does not match `great white shark` and the package installation proceeded fine, shouldn't the test die?

Comment (DeepthiYV) [reply tests/ai_ml/tflite2.pm:41]: Typically yes,  Test should fail if the python script(label_image.py) output does not match to the `great white shark`.  

For now I have temporarily added softfail as the python script failed to run due to known bug  (`failed to install python3-tensorflow`)

**Review by mloviska — APPROVED** [review]

File: tests/ai_ml/tflite2.pm (hunk 1 above)
Comment (DeepthiYV) [reply tests/ai_ml/tflite2.pm:43]: Thank you Martin for the feedback :)

**Review by Avinesh — APPROVED** [review]
LGTM

**Review by foursixnine — APPROVED** [review]

**Review by foursixnine — APPROVED** [review]

**Pull request merged**